docs/pages/image.js

```diff
@@ -9,53 +9,52 @@ import docgen, { type DocGen } from '../components/docgen.js';
 
 export default function DocsPage({ generatedDocGen }: {| generatedDocGen: DocGen |}): Node {
   return (
-    <Page title="Image">
-      <PageHeader name="Image" description={generatedDocGen?.description} />
+    <Page title={generatedDocGen?.displayName}>
+      <PageHeader name={generatedDocGen?.displayName} description={generatedDocGen?.description} />
 
       <GeneratedPropTable generatedDocGen={generatedDocGen} />
 
       <Card
         description={`
-    One thing that might be unusual is that the \`width\` and the \`height\` of the
-    component are required, yet the image will scale to the size of its container.
-    This is so that the placeholder's size can be calculated before the image has
-    rendered.
-
-    While the exact dimensions supplied aren't used (only the ratio between them is
-    considered), you should always try to try to supply the exact dimensions of the
-    source image requested.
+One thing that might be unusual is that the \`width\` and the \`height\` of the component are required, yet the image will scale to the size of its container.
+
+This is so that the placeholder's size can be calculated before the image has rendered.
+
+While the exact dimensions supplied aren't used (only the ratio between them is considered), you should always try to try to supply the exact dimensions of the source image requested.
   `}
         name="Dimensions"
       />
 
       <Example
         id="placeholders"
         description={`
-    The color you pass into \`Image\` will be used to fill the placeholder that shows up
-    as an image loads. The example shown has an empty \`src\` prop provided so it remains
-    a placeholder.
+The color you pass into \`Image\` will be used to fill the placeholder that shows up as an image loads. The example shown contains an image with transparent background which allows to visualize the placeholder color. Sometimes the placeholder color might be shown in the corners of an Image wrapped with Mask to provide rounding. To prevent this, remove the placeholder color.
```

Comment:
maybe: "he color you pass into \`Image\` will be used to fill the placeholder that shows up as an image loads. The example shown contains an image with a transparent background which allows you to visualize the placeholder color. When Mask is used to create rounded corners on Image, the placeholder color may leak through on the corners. To prevent this, remove the placeholder color."

Reply:
DONE